setup.py

```diff
@@ -43,7 +43,7 @@
         "Programming Language :: Python :: 3.10",
     ],
     # test_suite="tests",
-    install_requires=["helics-apps>=3.2.1", "helics>=3.2.1", "pydantic", "psutil", "click"],
+    install_requires=["helics-apps>=3.2.1", "helics>=3.2.1", "pydantic==1.10", "psutil", "click"],
```

Comment:
Is there a reason to tie to a specific version of pydantic?

Reply:
Leaving it unspecified ended up going to pydantic 2, which has several breaking changes. The older versions of pydantic handle forward references in a different way. I'll try and narrow it down.

Reply:
Fixed. It seems versions >= 1.7 work and the newest 2.0a... versions fail.